[2021-07-05 08:44:16,031] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: get_as_dataframe.write_diseases 2021-07-05T08:41:49.353787+00:00 [queued]>
[2021-07-05 08:44:16,056] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: get_as_dataframe.write_diseases 2021-07-05T08:41:49.353787+00:00 [queued]>
[2021-07-05 08:44:16,057] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-05 08:44:16,058] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2021-07-05 08:44:16,059] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-05 08:44:16,076] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): write_diseases> on 2021-07-05T08:41:49.353787+00:00
[2021-07-05 08:44:16,081] {standard_task_runner.py:52} INFO - Started process 2791 to run task
[2021-07-05 08:44:16,100] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_as_dataframe', 'write_diseases', '2021-07-05T08:41:49.353787+00:00', '--job-id', '1134', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_as_Dataframe.py', '--cfg-path', '/tmp/tmpkjk_077x', '--error-file', '/tmp/tmpvb__ssst']
[2021-07-05 08:44:16,103] {standard_task_runner.py:77} INFO - Job 1134: Subtask write_diseases
[2021-07-05 08:44:16,277] {logging_mixin.py:104} INFO - Running <TaskInstance: get_as_dataframe.write_diseases 2021-07-05T08:41:49.353787+00:00 [running]> on host airflowvm
[2021-07-05 08:44:16,439] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_as_dataframe
AIRFLOW_CTX_TASK_ID=write_diseases
AIRFLOW_CTX_EXECUTION_DATE=2021-07-05T08:41:49.353787+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-05T08:41:49.353787+00:00
[2021-07-05 08:44:16,441] {python.py:151} INFO - Done. Returned value was: None
[2021-07-05 08:44:16,476] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=get_as_dataframe, task_id=write_diseases, execution_date=20210705T084149, start_date=20210705T084416, end_date=20210705T084416
[2021-07-05 08:44:16,543] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-05 08:44:16,591] {local_task_job.py:151} INFO - Task exited with return code 0
